### DISCUSSION TITLE
Updating game.db to nice readable game names in next release of GT4286Util

### DISCUSSION THREAD

**Discussion by freeb1e4me** [nk64/GT4286#16] [Ideas]
I'm not sure what is planned for the next release of GT4286Util, but I would like to suggest:

-Updating game.db to nice readable game names

One option is to use the XML file, generated by scraper (https://github.com/sselph/scraper), to get the nice readable names for the games. Parse the XML file to match and replace with the new names and update game.db accordingly. What are people's thoughts on this. Cheers

**Comment by nk64** [comment]
All emulators other than FBNeo/MAME will work with whatever name the rom is given.
The names of those roms will come down to personal choice or whatever rom management tool you choose to use to curate your rom collection and are out of scope for GT4286Util. I use [RomVault](https://www.romvault.com/).

I have a good source of names for the FBNeo arcade roms in the dat file that is generated by ```/emus/fbneo/fbneo```. It is also XML and easy parse.

**Comment by nk64** [comment]
I have written up the manual process of [Humanising](https://github.com/nk64/GT4286/blob/main/Humanise%20Names.md) FBNeo arcade game names for the extra keen.

**Reply by ot644** [reply]
This is a good option for some.  Others such as myself have far too many roms in fba. Looking forward to an automated process in future releases of GT4286Util if possible but looks like it will be a challenge.

**Comment by freeb1e4me** [comment]
Thanks for the writeup on humanising FBNeo/Mame arcade game names. Regarding steps 1 & 2:

1. Rename the FBNeo emulator by adding .bin to the filename eg. /emus/fbneo/fbneo becomes /emus/fbneo/fbneo.bin

2. Replace your FBNeo emulator binary (/emus/fbneo/fbneo) with the shell script called fbneo.script_with_rom_name_redirection found [here](https://github.com/nk64/GT4286/blob/main/sdcard_tweaks/emus/fbneo) (rename it to fbneo of course)

Why rename fbneo to fbneo.bin if it's going to be **replaced** with a shell script fbneo? If you're just renaming the shell script called fbneo.script_with_rom_name_redirection to fbneo and not fbneo.bin then you're not replacing either. Am I reading the steps correctly?

**Reply by nk64** [reply]
Yeah, I felt a little weird as I was writing it. 

The end result is that you *must* have two files:
- ```/emus/fbneo/fbneo.bin``` (The real emulator)
- ```/emus/fbneo/fbneo``` (The shell script, that does some manipulation of the rom parameter and then calls the real emulator)

So Step 1 is correct and required, and maybe you can give me some wording to improve Step 2.

**Comment by freeb1e4me** [comment]
How about something like this:

Step 2: Place the shell script found [here](https://github.com/nk64/GT4286/blob/main/sdcard_tweaks/emus/fbneo) in /emus/fbneo/fbneo.script_with_rom_name_redirection then rename script to fbneo.

**Reply by nk64** [reply]
I have updated the instructions a bit to clarify.
